test/browser/features/steps/browser_steps.rb

```diff
@@ -41,3 +41,15 @@
   txt = Maze.driver.find_element(id: 'bugsnag-test-state').text
   assert_equal('DONE', txt, "Expected #bugsnag-test-state text to be 'DONE'. It was '#{txt}'.")
 end
+
+Then("the event {string} has no variant") do |field_path|
```

Comment:
As mentioned above we should add some steps to maze-runner core to perform these steps unless they're going to seriously vary between each notifier.